### PR TITLE
Adding workflow automation for spark-rapids-jni [skip ci]

### PR DESCRIPTION
**Pull request by mattahrens**
Signed-off-by: Matt Ahrens <matthewahrens@gmail.com>

Automatically adds new issues into global spark-rapids Github Project

### CODE REVIEW

File: .github/workflows/add-to-project.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+jobs:
+  add-to-project:
+    if: github.repository == 'NVIDIA/spark-rapids-jni'
+    name: Add new issues to project
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/add-to-project@v0.3.0
+        with:
+          project-url: https://github.com/orgs/NVIDIA/projects/4
```
Comment (pxLi) [inline .github/workflows/add-to-project.yml:30]: Q:  so we could add issues from different repos to the same github project, right?

**Review by pxLi — APPROVED** [review]
LGTM, just one question

**Comment by pxLi** [comment]
build

File: .github/workflows/add-to-project.yml (hunk 1 above)
Comment (mattahrens) [reply .github/workflows/add-to-project.yml:30]: Yes.  That's the benefit of the new GitHub Projects capability, in that it allows us to bring together issues across repos for a single view.

**Pull request merged**